server/pulp/server/webservices/views/serializers/link.py

```diff
@@ -0,0 +1,59 @@
+"""
```

Comment:
Was this code present elsewhere? If so then a `git mv` would let git know that the lines came from the original file. If that is the case you could remove/add after the `git mv` has taken place. If you wrote all this from scratch then disregard my comment.